x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/index.tsx

```diff
@@ -39,14 +45,42 @@ import {
   policyResponseAppliedRevision,
 } from '../../store/selectors';
 import { EndpointDetails } from './endpoint_details';
+import { EndpointActivityLog } from './endpoint_activity_log';
 import { PolicyResponse } from './policy_response';
+import * as i18 from '../translations';
 import { HostMetadata } from '../../../../../../common/endpoint/types';
+import {
+  EndpointDetailsFlyoutTabs,
+  EndpointDetailsTabsTypes,
+} from './components/endpoint_details_tabs';
+
 import { PreferenceFormattedDateFromPrimitive } from '../../../../../common/components/formatted_date';
 import { EndpointIsolateFlyoutPanel } from './components/endpoint_isolate_flyout_panel';
 import { BackToEndpointDetailsFlyoutSubHeader } from './components/back_to_endpoint_details_flyout_subheader';
 import { FlyoutBodyNoTopPadding } from './components/flyout_body_no_top_padding';
 import { getEndpointListPath } from '../../../../common/routing';
 
+const contentLoadingMarkup = (
+  <>
+    <EuiLoadingContent lines={3} /> <EuiSpacer size="l" /> <EuiLoadingContent lines={3} />
+  </>
+);
+
+const DetailsFlyoutBody = styled(EuiFlyoutBody)`
```

Comment:
Why did we need to adjust the flyout body?
I thought it already handled scrolling properly including when header/footers are included/excluded.

Reply:
So this we need in order to ensure that the tabs within the body do not scroll and only the tab content scrolls. This whole tab component within the flyout needs to be done better using [flyout#more-complicated-flyout](https://elastic.github.io/eui/#/layout/flyout#more-complicated-flyout) version of the flyout component. I would do that in the new PR when I take care of pagination UX.